dev/io.openliberty.webcontainer.servlet.6.0.internal/src/io/openliberty/webcontainer60/osgi/container/config/WebAppConfiguratorHelper60.java

```diff
@@ -1,14 +1,11 @@
 /*******************************************************************************
- * Copyright (c) 2022 IBM Corporation and others.
+ * Copyright (c) 2023 IBM Corporation and others.
```

Comment:
Why was the date changed and a range not used ?

Reply:
updated